aptos-move/framework/aptos-framework/sources/stake.spec.move

```diff
@@ -0,0 +1,73 @@
+spec aptos_framework::stake {
```

Comment:
Can you add some comments here to explain the different ensures, esp complex ones?

Reply:
I added some. More will come.